charts/partials/templates/_volumes.tpl

```diff
@@ -3,3 +3,12 @@ emptyDir:
   medium: Memory
 name: linkerd-identity-end-entity
 {{- end -}}
+
```

Comment:
I think it would be very helpful to have some comments here describing what this is and the context for it existing.